.github/workflows/release.yml

```
@@ -21,9 +21,14 @@ jobs:
         run: |
           git config user.name "$GITHUB_ACTOR"
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+     
+     - name: Add helm dependency
+        run: |
+          helm repo add bitnami https://charts.bitnami.com/bitnami
 
       - name: Install Helm
         uses: azure/setup-helm@v3
+        token: ${{ secrets.GITHUB_TOKEN }}
```

The example in https://github.com/Azure/setup-helm/blob/ac5ee1fca8fba06d82bfc39af726f19ffe037a49/README.md?plain=1#L10C12-L10C12 suggest this notation:

```suggestion
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
```

I think the `with` is mandatory here.